trinity/rpc/modules/net.py

```diff
@@ -10,3 +10,21 @@ def version(self):
         Returns the current network ID.
         """
         return str(self._chain.network_id)
+
+    def peerCount(self):
+        """
+        Return the number of peers that are currently connected to the node
+        """
+        if self._p2p_server.peer_pool is None:
+            return '0x0'
+
+        return hex(self._p2p_server.peer_pool.connected_peer_count)
+
+    def listening(self):
+        """
+        Return `True` if the client is actively listening for network connections
+        """
+        if self._p2p_server.peer_pool is None:
+            return False
+
+        return not self._p2p_server.peer_pool.is_full
```

Comment:
Are we sure this is the right logic.  We are technically still listening (have an outbound socket open that accepts incoming connections) even when the peer pool is full.  I think the `discovery` service is where we'd need to look to determine if we're listening or not.

Reply:
I'm gonna double check that. Here is what pyethapp does which isn't the same as what we do but not that far away either :sweat_smile: 

https://github.com/ethereum/pyethapp/blob/9af6e64e6140104ab13fdbc5a6b22a915c412adc/pyethapp/jsonrpc.py#L696

Reply:
> `// (this may not be true now -- 26 september 2016)`

lol